diff --git a/node_modules/@pancakeswap/uikit/dist/index.esm.js b/node_modules/@pancakeswap/uikit/dist/index.esm.js
index 2545523..86e287e 100644
--- a/node_modules/@pancakeswap/uikit/dist/index.esm.js
+++ b/node_modules/@pancakeswap/uikit/dist/index.esm.js
@@ -4341,20 +4341,24 @@ var Pip = styled.div(templateObject_2$1 || (templateObject_2$1 = __makeTemplateO
     return theme.colors.failure;
 });
 var Avatar = function (_a) {
+    
     var profile = _a.profile;
     var _b = profile.username, username = _b === void 0 ? "Bunny" : _b, image = profile.image, profileLink = profile.profileLink, noProfileLink = profile.noProfileLink, _c = profile.showPip, showPip = _c === void 0 ? false : _c;
     var link = profile.username ? profileLink : noProfileLink;
     var isExternal = link.startsWith("http");
     var ariaLabel = "Link to profile";
     var icon = image ? (React.createElement("img", { src: image, alt: "profile avatar", height: "32px", width: "32px" })) : (React.createElement(Icon$11, { width: "32px", height: "32px" }));
+    
     if (isExternal) {
         return (React.createElement(StyledAvatar, { title: username },
             React.createElement("a", { href: link, "aria-label": ariaLabel }, icon),
             showPip && React.createElement(Pip, null)));
     }
+    
     return (React.createElement(StyledAvatar, { title: username },
         React.createElement(Link$1, { to: link, "aria-label": ariaLabel }, icon),
         showPip && React.createElement(Pip, null)));
+
 };
 var templateObject_1$4, templateObject_2$1;
 
@@ -4426,8 +4430,8 @@ var Menu = function (_a) {
         React.createElement(StyledNav, { showMenu: showMenu },
             React.createElement(Logo$1, { isPushed: isPushed, togglePush: function () { return setIsPushed(function (prevState) { return !prevState; }); }, isDark: isDark, href: (_b = homeLink === null || homeLink === void 0 ? void 0 : homeLink.href) !== null && _b !== void 0 ? _b : "/" }),
             !!login && !!logout && (React.createElement(Flex, null,
-                React.createElement(UserBlock$1, { account: account, login: login, logout: logout }),
-                profile && React.createElement(Avatar, { profile: profile })))),
+                React.createElement(UserBlock$1, { account: account, login: login, logout: logout })/*,
+                 profile && React.createElement(Avatar, { profile: profile }) */))),
         React.createElement(BodyWrapper, null,
             React.createElement(Panel, { isPushed: isPushed, isMobile: isMobile, showMenu: showMenu, isDark: isDark, toggleTheme: toggleTheme, langs: langs, setLang: setLang, currentLang: currentLang, cakePriceUsd: cakePriceUsd, pushNav: setIsPushed, links: links }),
             React.createElement(Inner, { isPushed: isPushed, showMenu: showMenu }, children),
